test/e2e/cassandracluster_test.go

```diff
@@ -144,3 +150,124 @@ func cassandraClusterRollingRestartDCTest(t *testing.T, f *framework.Framework,
 
 	assert.Equal(t, false, cc.Spec.Topology.DC[0].Rack[0].RollingRestart)
 }
+
+func cassandraClusterInitMultiDCTest(t *testing.T, f *framework.Framework, ctx *framework.TestCtx) {
+	namespace, err := ctx.GetNamespace()
+	if err != nil{
+		t.Fatalf("Could not get namespace: %v", err)
+	}
+
+	cluster := &api.CassandraCluster{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "CassandraCluster",
+			APIVersion: "db.orange.com/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "multidc-cluster",
+			Namespace: namespace,
+			Labels:    map[string]string{"cluster": "k8s.pic"},
+		},
+		Spec: api.CassandraClusterSpec{
+			BaseImage: "orangeopensource/cassandra-image",
+			NodesPerRacks: 2,
+			HardAntiAffinity: false,
+			DeletePVC: true,
+			Resources: api.CassandraResources{
+				Limits: api.CPUAndMem{
+					CPU: "500m",
+					Memory: "1Gi",
+				},
+			},
+			Topology: api.Topology{
```

Comment:
Please try to use the minimum required for your tests in order to minimize the time e2e tests take.
For example, using 1 node per rack and only one rack per DC here.

Reply:
Because the e2e tests are expensive to run, I kind of felt like it would be overkill to have the test only verify that a single service is created for the cluster. I figured I might as well verify that other objects, namely StatefulSet, get created. 

While I think what I described would be a good test to have, I am fine with using the minimum resources necessary and leaving a multi-DC/multi-rack scenario for a separate test. Would that be better?

Reply:
Yeah I think so. Tests are gonna take longer the more we add in here. Trying to limit the number of times we need to instantiate a cluster (which takes time) makes more sense to me.